src/preact/base-element.js

```diff
@@ -852,20 +853,12 @@ export class PreactBaseElement extends AMP.BaseElement {
     }
     this.isPlaying_ = isPlaying;
     if (isPlaying) {
-      observeContentSize(this.element, this.pauseWhenNoSize_);
+      if (!this.pauseHelper_) {
+        this.pauseHelper_ = new PauseHelper(this.element);
```

Comment:
Nit: can we just initialize in `constructor`?

Reply:
We can. You ok with an extra allocation? Since in many cases it won't be needed until the first playback is detected in an element.

Reply:
I think it's fine, Bento aims for deferred build anyways so the constructor is lazy.

Reply:
Done